client/src/plugins/zeebe-plugin/deployment-plugin/DeploymentPlugin.js

```diff
@@ -37,6 +37,8 @@ import DeploymentPluginValidator from './DeploymentPluginValidator';
 
 const DEPLOYMENT_CONFIG_KEY = 'zeebe-deployment-tool';
 
+const ET_EXECUTION_PLATFORM_NAME = 'Camunda Cloud';
```

Comment:
This assumes that all Zeebe deployments go to Camunda Cloud. How about self-hosted Zeebe instances?

Reply:
For me `Camunda Cloud` and `Camunda Platform` are the new to-be-used names for `Camunda Runtime` and `Zeebe` respectively. @volkergersabeck correct me if wrong.

Note that there is still the [target type field](https://github.com/camunda/camunda-modeler/blob/develop/client/src/plugins/usage-statistics/event-handlers/DeploymentEventHandler.js#L132) which would either be `camundaCloud` or `selfHosted`. So can still differenciate that.

Reply:
Update: I talked to @volkergersabeck. He will think about this considering also different places that we have, and get back to us.

Reply:
@barmac (also see slack #modeling-team). Following the discussion with volker using product names is the desired state. So this should be fine